lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -0,0 +1,81 @@
+.container {
+    background-color: #fff;
+    display: inline-flex;
+    flex-direction: column;
+
+    button,
+    svg {
+        color: #337ab7;
+
+        &:focus,
+        &:hover {
+            background: #efefef;
+            color: #000;
```

Comment:
Should use the color variable `$color-link-black` for this.

Reply:
Here as well.